openSUSE/open-build-service
# Add theming support

This is a simple theming implementation, with a sample openSUSE theme (which shouldn't be used in production, I will leave it up to @guoyunhe to implement the remaining bits and pieces)

To verify this feature

- Change theme variable in config from `default` to `openSUSE`
- See a different look of OBS

No screenshots because you really shouldn't use the openSUSE theme yet

Fixes #6311
